src/os/bluestore/BlueStore.h

```diff
@@ -2209,6 +2210,9 @@ class BlueStore : public ObjectStore,
   */
   int _open_db_and_around(bool read_only);
   void _close_db_and_around();
+  int cold_open();
```

Comment:
make it public and remove the trick with friend class?

Reply:
Done.